docs/drafts/derive-serviceaccount.md

```diff
@@ -0,0 +1,355 @@
+# Derive minimal ServiceAccount required for ClusterExtension Installation and Management
+
+Adhering to OLM v1's "Secure by Default" tenet, OLM v1 does not have the permissions necessary to install content. This follows the least privilege principle and reduces the chance of a [confused deputy attack](https://en.wikipedia.org/wiki/Confused_deputy_problem). Instead, a ServiceAccount must be provided by users to install and manage content.
```

Comment:
Using the same blocks of text as in the other document seems repetitive. Do we think these should be combined or should we reconfigure this opening section to be more focused directly on how to determine the required RBAC permissions for the SA?

Reply:
i think i will remove the opening block and dive straight into the required rbac